monitor/alerts/list-get-example-all/list-get-example-all.3.x.js

```diff
@@ -0,0 +1,13 @@
+// Download the Node helper library from twilio.com/docs/node/install
+// These are your accountSid and authToken from https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
+
+const MonitorClient = require('twilio').MonitorClient;
+const client = new MonitorClient(accountSid, authToken);
+
+client.alerts.list(function(err, data) {
```

Comment:
this should be used instead: https://github.com/twilio/twilio-node/blob/next-gen/lib/rest/monitor/v1/alert.js#L130

similar to: https://github.com/TwilioDevEd/api-snippets/blob/next-gen-node-snippets-rest-part-1/rest/call/list-get-example-2/list-get-example-2.3.x.js#L9